pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -230,7 +230,7 @@ def configure_progress_bar(self, refresh_rate=None, process_position=0):
         if len(progress_bars) == 1:
             progress_bar_callback = progress_bars[0]
         elif refresh_rate > 0:
-            progress_bar_callback = ProgressBar(refresh_rate=refresh_rate, process_position=process_position)
+            progress_bar_callback = TQDMProgressBar(refresh_rate=refresh_rate, process_position=process_position)
```

Comment:
here too: https://github.com/PyTorchLightning/pytorch-lightning/blob/2d4d60934d1bb7ffcda5995ae7706c508e2ae64c/pytorch_lightning/trainer/connectors/callback_connector.py#L82-L95
here too in the same file